cmd/clusterctl/internal/util/yaml.go

```diff
@@ -53,46 +48,6 @@ func JoinYaml(yamls ...[]byte) []byte {
 	return r
 }
 
-// ToUnstructured takes a YAML and converts it to a list of Unstructured objects
-func ToUnstructured(rawyaml []byte) ([]unstructured.Unstructured, error) {
```

Comment:
Given that we are moving this `ToUnstructured `, I think we should also `FromUnstructured`. WDYT?

Reply:
If that's the case we should just move this entire package because `FromUnstructured` calls `JoinYaml`.

Reply:
I think these are all generic, we can move them.
Also there is yamlToUnstructured() in KCP code which can be moved to utils too or we can use ToUnstructured for that too.